lab-michelle/lib/read-file.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const fs = require('fs');
+//Requiring in the file system
+
+console.log(`current directory: ${__dirname}`);
+
+let x = module.exports = {};
+
+x.read = (callback) => {
+
+  let first, second, third;
+
+  fs.readFile(`${__dirname}/../assets/one.txt`,(err, data) => {
```

Comment:
refactor these based on lecture demo today so they are nested. otherwise you won't know if they are actually returning in order. part of the assignment also included printing the first 8 bytes to the console.